pubsub/feature.go

```diff
@@ -25,11 +29,5 @@ type Feature string
 
 // IsPresent checks if a given feature is present in the list.
 func (f Feature) IsPresent(features []Feature) bool {
-	for _, feature := range features {
-		if feature == f {
-			return true
-		}
-	}
-
-	return false
+	return slices.Contains(features, f)
```

Comment:
Why was this modified? This kind of change can also be the cause of subtle bugs.

Also, you are using the experimental slices packages. Maybe best to avoid `golang.org/x/exp`

Reply:
Was requested by @mukundansundar here: https://github.com/dapr/components-contrib/pull/2237#discussion_r1142498284

Not concerned about it being "experimental" however. It's in `exp` because it uses generics, but otherwise it's exactly the same code we had before. We use it in other parts of Dapr.

Anyways, i'm happy to revert if you'd like.

Reply:
@mukundansundar in the future let's be consistent. Refactoring this to using `slices` could have been elsewhere.